internal/sso/get/get.go

```diff
@@ -0,0 +1,46 @@
+package get
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/supabase/cli/internal/sso/internal/render"
+	"github.com/supabase/cli/internal/utils"
+	"github.com/supabase/cli/pkg/api"
+)
+
+func Run(ctx context.Context, ref, providerId, format string) error {
+	resp, err := utils.GetSupabase().GetProviderByIdWithResponse(ctx, ref, providerId)
+	if err != nil {
+		return err
+	}
+
+	if resp.JSON200 == nil {
+		if resp.StatusCode() == http.StatusNotFound {
+			return fmt.Errorf("An identity provider with ID %q could not be found.", providerId)
+		}
+
+		return errors.New("Unexpected error fetching identity provider: " + string(resp.Body))
+	}
+
+	switch format {
+	case "metadata":
```

Comment:
Let's make a new constant for this special case in this file, ie. `const OutputMetadata = "metadata"`